svgpathtools/paths2svg.py

```diff
@@ -141,58 +140,38 @@ def disvg(paths=None, colors=None,
         :param mindim - The minimum dimension (height or width) of the output
         SVG (default is 600).
 
-        :param dimensions - The (x,y) display dimensions of the output SVG.
```

Comment:
Looks like you're trying to replace new/improved comments with their old versions here.

Reply:
Like in line 92? Yep... (Not my changes.)